test/test_tracker.py

```diff
@@ -10,15 +10,15 @@
 
 
 class TestHPyTracker(HPyTest):
-    def hpytracker_module(self, ops, new="HPyTracker_New(ctx, 0)"):
+    def hpytracker_module(self, ops, size=0):
```

Comment:
The `size` change is unrelated to the argument parsing, but @antocuni complained that passing more code here made finding the code that was creating the tracker complicated, so I improved this while I was in the file.